pepper-apis/src/main/java/org/broadinstitute/ddp/json/PatchLastVisitedSectionPayload.java

```diff
@@ -0,0 +1,17 @@
+package org.broadinstitute.ddp.json;
+
+import javax.validation.constraints.PositiveOrZero;
+
+public class PatchLastVisitedSectionPayload {
+
+    @PositiveOrZero
+    private int lastVisitedSection;
```

Comment:
OK! Looks like you are looking for zeros and positive values. So assuming you want Java/JavaScript-style array index position